com.github.AberothCommunity.Aberoth.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.AberothCommunity.Aberoth</id>
+  <name>Aberoth</name>
+  <developer_name>Aberoth</developer_name>
+  <summary>Free 8-bit MMORPG</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://aberoth.com/help.php</project_license>
```

Comment:
Is redistribution allowed? Did you contact them?

Reply:
Yes, I sent him an email.

Reply:
it uses extra-data so redistribution is not the problem